swarm/api/config_test.go

```diff
@@ -27,11 +27,16 @@ import (
 func TestConfig(t *testing.T) {
 
 	var hexprvkey = "65138b2aa745041b372153550584587da326ab440576b2a1191dd95cee30039c"
+	var hexnodekey = "75138b2aa745041b372153550584587da326ab440576b2a1191dd95cee30039c"
 
 	prvkey, err := crypto.HexToECDSA(hexprvkey)
 	if err != nil {
 		t.Fatalf("failed to load private key: %v", err)
 	}
+	nodekey, err := crypto.HexToECDSA(hexnodekey)
```

Comment:
this is always there whenever you call config.Init() could we not just put the nodkey generation into `Init`?

Reply:
@zelig no because we want to generate it with `node/Config.NodeKey()` when in `cmd/swarm`